javers-core/src/main/java/org/javers/repository/api/QueryParamsBuilder.java

```diff
@@ -89,11 +90,19 @@ public QueryParamsBuilder commitId(CommitId commitId) {
         return this;
     }
 
+    /*
+     * limits results to Snapshots with a given version
+     */
+    public QueryParamsBuilder version(Long version) {
```

Comment:
imo we should use `long` not `Long` here, the same as in `CdoSnapshot.getVersion()`